app/View/Label.php

```diff
@@ -116,12 +116,9 @@ public function render(callable $callback = null)
                         }
                     }
               
-                if ($template->getSupport2DBarcode()) {
+            if ($template->getSupport2DBarcode()) {
                     $barcode2DType = $settings->label2_2d_type;
-                    $barcode2DType = ($barcode2DType == 'default') ? 
```

Comment:
It is still possible for `default` to be what is set in existing databases right? I don't think we want to remove this check if so.

Reply:
Note: @Godmartinz pointed out that `default` was changed in a migration in #16033.